buildAndReleaseTask/pulumi.ts

```diff
@@ -222,15 +222,6 @@ export async function runPulumi() {
             ...processEnv,
         };
 
-        // For DotNet projects, the dotnet CLI requires a home directory (sort of a temp directory).
-        // On Azure Pipelines, the user home env var is undefined, and the workaround is to
-        // set the DOTNET_CLI_HOME env var. This is not a Pulumi-specfic env var.
-        const dotnetCliHome =
```

Comment:
@JasonWhall is the expectation that users must set this on their own for the case of using the pre-built Pulumi binary option? Or is this never required now that this task extension reads all of the build agent's env vars, which is what I understood from https://github.com/pulumi/pulumi-az-pipelines-task/issues/40#issuecomment-722207285? If so, we should add it to the [`overview.md`](https://github.com/pulumi/pulumi-az-pipelines-task/blob/master/overview.md). Perhaps under a new section called "Troubleshooting tips"? Basically, something that informs users when to set that env var.

Reply:
@praneetloke - This should only be required if `dotnet` does not have access to the users home (Linux - `$HOME`/Windows - `$env:USERPROFILE`) directory. Azure DevOps agents have had access to System and Environment variables for a a long while now, so should not be needed..

I'll add something into `overview.md` to mention this behaviour as this should be an exception, rather than the standard to set this variable.

Reply:
Sounds good. Thank you!